### PR TITLE
New py3 dlc dockerfiles

### PR DESCRIPTION
**Pull request by saimidu**
Create Dockerfile.eia for 1.4.1 with py3, with health check binary included

Tests

- Tested by serving squeezenet model on container.
- Tested with Sagemaker integration tests - "batch_transform.py", " 	test_elastic_inference.py", "test_hosting.py".

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/1.4.1/py3/Dockerfile.eia [hunk 1]
```diff
@@ -1,22 +1,56 @@
...
+
+LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.8
+ARG HEALTH_CHECK_VERSION=1.3.2
```
Comment (surajkota) [inline docker/1.4.1/py3/Dockerfile.eia]: does this need to be updated to 1.3.3. I see in TF dockerfiles

File: docker/1.4.1/py3/Dockerfile.eia [hunk 2]
```diff
@@ -1,22 +1,56 @@
-FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py3.6
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python3
+ARG PIP=pip3
```
Comment (mvsusp) [inline docker/1.4.1/py3/Dockerfile.eia]: Pip should be determined by the python version

Comment (saimidu) [reply docker/1.4.1/py3/Dockerfile.eia]: Removed this ARG

File: docker/1.4.1/py3/Dockerfile.eia [hunk 3]
```diff
@@ -1,22 +1,56 @@
...
 
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
+    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
+    ./configure && make && make install && \
+    apt-get update && apt-get install -y --no-install-recommends libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev && \
+    make && make install && rm -rf ../Python-$PYTHON_VERSION* && \
+    ln -s /usr/local/bin/pip3 /usr/bin/pip
```
[the remark this answers is not in the document]
Comment (saimidu) [reply docker/1.4.1/py3/Dockerfile.eia]: Improved indentation.

File: docker/1.4.1/py3/Dockerfile.eia [hunk 4]
```diff
@@ -1,22 +1,56 @@
...
+
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
```
[the remark this answers is not in the document]
Comment (saimidu) [reply docker/1.4.1/py3/Dockerfile.eia]: Removed usage of ARGs PYTHON and PIP after alias is done.

**Pull request merged**